### PR TITLE
Add Urdu Latin Sentiment Classification Dataset

### PR DESCRIPTION
**Pull request by bp-high**
## Checklist for adding MMTEB dataset
Reason for dataset addition:
- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [ ] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

**Review by asparius — COMMENTED** [review]
Looks fine, but same usage of train split instead  of eval split as in #536.

**Review by KennethEnevoldsen — COMMENTED** [review]
Minor comment here as well

File: mteb/tasks/Classification/urd/UrduRomanSentimentClassification.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        @misc{misc_roman_urdu_data_set_458,
+  author       = {Sharf,Zareen},
+  title        = {{Roman Urdu Data Set}},
+  year         = {2018},
+  howpublished = {UCI Machine Learning Repository},
+  note         = {{DOI}: https://doi.org/10.24432/C58325}
+}
+    """,
+        n_samples={"train": 20229},
+        avg_character_length=None,
```
[the remark this answers is not in the document]
Comment (asparius) [reply mteb/tasks/Classification/urd/UrduRomanSentimentClassification.py:40]: i.e. np.average(list(map(len,data["train"]["text"))) should work for avg_character_length @bp-high

File: mteb/tasks/Classification/urd/UrduRomanSentimentClassification.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        bibtex_citation="""
+        @misc{misc_roman_urdu_data_set_458,
+  author       = {Sharf,Zareen},
+  title        = {{Roman Urdu Data Set}},
+  year         = {2018},
+  howpublished = {UCI Machine Learning Repository},
+  note         = {{DOI}: https://doi.org/10.24432/C58325}
+}
+    """,
+        n_samples={"train": 20229},
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/urd/UrduRomanSentimentClassification.py:39]:
```suggestion
        n_samples={"train": 2048},
```

**Review by KennethEnevoldsen — COMMENTED** [review]
Looks good feel free to add points and the change above then we can merge

File: mteb/tasks/Classification/urd/UrduRomanSentimentClassification.py (hunk 2 above)
Comment (bp-high) [reply mteb/tasks/Classification/urd/UrduRomanSentimentClassification.py:39]: Okay. For the points what should be the values and I have to give points to both reviewers right?

Comment (KennethEnevoldsen) [reply mteb/tasks/Classification/urd/UrduRomanSentimentClassification.py:39]: We give 2 points for the dataset and +4 bonus points for languages not previously covered in classification. review points should be given to all reviewers.

Comment (bp-high) [reply mteb/tasks/Classification/urd/UrduRomanSentimentClassification.py]: Cool added 2 points for dataset(as urdu already present in IndicLangClassification dataset [mteb/tasks/Classification/multilingual/IndicLangClassification.py](https://github.com/embeddings-benchmark/mteb/blob/cf1c8d19bdbc8984c367ff74d435e66a8ccc61a9/mteb/tasks/Classification/multilingual/IndicLangClassification.py#L35) and 2 points for each reviewer.

**Pull request merged**